introduction.md

```diff
@@ -2,7 +2,10 @@ Hello, my name is Dean and I've come a long way from knowing nothing about softw
 
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?